src/app.rs

```diff
@@ -1166,6 +1526,36 @@ impl UiExtra for egui::Ui {
     }
 }
 
+fn get_entries_with_level<'a>(items: &Vec<&'a EntryID>, level: u64) -> Vec<Vec<&'a EntryID>> {
```

Comment:
I'm trying to understand this code and am not sure that `level` is an accurate name. What's this doing?